docs/accessing-data/connecting.mdx

```diff
@@ -0,0 +1,134 @@
+---
+title: "Connecting"
+sidebar_position: 10
+---
+
+BigQuery offers multiple connection methods to the Hubble dataset. This guide details three common methods:
+
+- [BigQuery UI](#bigquery-ui) - analysts that need to perform ad hoc analysis using SQL
+- [BigQuery SDK](#bigquery-sdk) - developers that need to integrate data into applications
+- [Looker Studio](#looker-studio) - business people that need to visualize data
+
+## Prerequisites
+
+To access the Hubble dataset, you will need a Google Cloud Project with billing and the BigQuery API enabled. For more information, please follow the instructions provided by [Google Cloud](https://cloud.google.com/bigquery/docs/quickstarts/query-public-dataset-console).
+
+Google does provide a BigQuery Sandbox for free that allows users to explore datasets in a limited capacity.
+
+## BigQuery UI
+
+1. From a browser, open the [crypto-stellar.crypto_stellar](http://console.cloud.google.com/bigquery?ws=!1m4!1m3!3m2!1scrypto-stellar!2scrypto_stellar) dataset.
+2. This will open the public dataset `crypto_stellar`, where you can browse its contents in the **Explorer** pane.
+3. Click the **star** icon in the Explorer pane. This will favorite the dataset for you. More detailed information about starring resources can be found [here](https://cloud.google.com/bigquery/docs/bigquery-web-ui#star_resources).
+
+:::note Hubble cannot be found from the Explorer pane! You cannot search for the dataset. To view the dataset, you **must** use the [dataset link](https://console.cloud.google.com/bigquery?ws=!1m4!1m3!3m2!1scrypto-stellar!2scrypto_stellar). :::
+
+Copy and paste the following example query in the Editor:
+
+<CodeExample>
+
+```sql
+select
+  account_id,
+  balance
+from `crypto-stellar.crypto_stellar.accounts_current`
+order by balance desc;
+```
+
+</CodeExample>
+
+This query will return the XLM balances for all Stellar wallet addresses, ordered from largest to smallest amounts.
+
+## BigQuery SDK
+
+There are multiple [BigQuery API Client Libraries](https://cloud.google.com/bigquery/docs/reference/libraries) available.
+
+The following example uses Python to access the Hubble dataset.
+
+Install the client library locally, and configure your environment to use your Google Cloud Project:
+
+<CodeExample>
+
+```bash
+pip install --upgrade google-cloud-bigquery
+gcloud config set project PROJECT_ID
```

Comment:
does this `pip` command add `gcloud` and friends to your PATH immediately? or do you need to e.g. restart shell, load some `.sh` file into your env, etc.? maybe providing a `.venv/bin/gcloud` (or equivalent) path would prevent silly "can't find `gcloud` pls halp???" questions

Reply:
great point, i'll add that